vite.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require("path");
+const { defineConfig } = require("vite");
+
+module.exports = defineConfig({
+  build: {
+    minify: false,
```

Comment:
Don't know Vite configs, but shouldn't build be minify true and sourcemap false?

Reply:
Does the use case of the library match one where it always makes sense to assume they will be using build tools?

Reply:
I'll let @lucaas answer that. :)

Reply:
But in general I believe that if [uuid](https://github.com/uuidjs/uuid/blob/main/CHANGELOG.md) can drop minified UMD builds, so can we. :)

Reply:
As for use case, I don't think a build/bundle step is necessarily expected but probably most common. We have examples using this library with and without bundling at https://help.ftrack.com/en/articles/1040449-custom-widgets . I guess ideally the UMD dist would be minified but the es module wouldn't. However, this library is small enough that it doesn't really matter.

It makes sense to align with the JS API client/SDK: https://github.com/ftrackhq/javascript-api/blob/main/vite.config.js#L7-L8

I am guessing that consumers that don't build/bundle doesn't care too much about it being minified.